www/public/index.html

```diff
@@ -8,26 +8,40 @@
   <title>The Roc Programming Language</title>
   <!-- <meta name="description" content="A language for making delightful software."> -->
   <meta name="viewport" content="width=device-width">
-  <style type="text/css">
-    html {
-      max-width: 35em;
-      margin: 0 auto;
-      font-family: sans-serif;
-      line-height: 145%;
-    }
-
-    li {
-      margin-bottom: 0.5rem;
-    }
-  </style>
+  <link rel="stylesheet" href="/homepage.css">
   <!-- <link rel="icon" href="/favicon.svg"> -->
 </head>
 
 <body>
-  <h1>Work in Progress</h1>
-  <p><b>Roc's initial release is still under development, and this website is a placeholder until that release is ready.
-  </b></p>
-  <p>In the meantime, if you'd like to learn more about Roc, here are some videos:</p>
+  <h1>The Roc Programming Language</h1>
+  <p>Roc's goal is to be a fast, friendly, functional language. It's very much a work in progress;
+    below, you can see the current progress towards this goal. This website is intentionally unstyled
+    as a way to emphasize the language's current level of incompleteness. The website will become
+    more polished after the language itself becomes more polished!</p>
+
+  <p>Roc compiles to machine code or to <a href="https://webassembly.org">WebAssembly</a>. Eventually
+    you'll be able to use Roc to build high-quality servers, command-line applications, or graphical
+    native desktop user interfaces. Today, only command-line interfaces have support beyond the
```

Comment:
is this the exhaustive niche intended for Roc, or would it make sense to say something like
```suggestion
    you'll be able to use Roc to build high-quality servers, command-line applications, graphical
    native desktop user interfaces, among other classes of applications. Today, only command-line interfaces have support beyond the
```